packit/source_git.py

```diff
@@ -152,9 +152,15 @@ def _run_prep(self):
 
     def get_BUILD_dir(self):
         path = Path(self.dist_git.working_dir)
-        build_dirs = [d for d in (path / "BUILD").iterdir() if d.is_dir()]
+        build_dirs = [
+            d
+            for d in (path / "BUILD").iterdir()
+            if d.is_dir() and not d.name.endswith("-SPECPARTS")
```

Comment:
I know it's still a draft..;) But can you please comment and/or mention in the commit why we are doing this?..;)

Reply:
I was planning to do so ;D but it is still in stage of testing and not ready for reviews, therefore draft